### PR TITLE
Lt 2411 maritime antipiracy

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: applications/helpers/check_your_answers.py [hunk 1]
```diff
@@ -246,6 +253,15 @@ def _get_route_of_goods(application):
     ]
 
 
+def _get_goods_categories(application):
+    return [
+        {
+            "Description": "Contains firearm products",
```
Comment (rorysproule1) [inline applications/helpers/check_your_answers.py:259]: Move to content?

File: ui_automation_tests/pages/apply_for_a_licence_page.py [hunk 2]
```diff
@@ -52,6 +46,9 @@ def select_types_of_clearance(self):
     def click_permanent_or_temporary_button(self, string):
         self.driver.find_element_by_id(self.EXPORT_BUTTON + string).click()
 
+    def select_firearms_yes(self):
+        self.driver.find_element_by_css_selector("[id$=-True]").click()
```
Comment (rorysproule1) [inline ui_automation_tests/pages/apply_for_a_licence_page.py:50]: is it possible to use id here?

File: applications/helpers/check_your_answers.py [hunk 3]
```diff
@@ -122,9 +118,20 @@ def _convert_standard_application(application, editable=False, is_summary=False)
...
+
+    return converted_app
+
 
 def _convert_open_application(application, editable=False):
     return {
+        **(
+            {applications.ApplicationSummaryPage.GOODS_CATEGORIES: _get_goods_categories(application),}
+            if application.case_type["reference"]["key"] == "oiel"
+            and application.goodstype_category["key"] in ["military", "uk_continental_shelf"]
```
Comment (Mark-Spiers-kainos) [inline applications/helpers/check_your_answers.py:132]: should be in a constant

File: apply_for_a_licence/forms/triage_questions.py [hunk 4]
```diff
@@ -65,6 +65,11 @@ def opening_question():
 
 
 def export_licence_questions(application_type, goodstype_category=None):
+    should_display_firearms_question = application_type == CaseTypes.SIEL or goodstype_category in [
+        "military",
```
Comment (Mark-Spiers-kainos) [inline apply_for_a_licence/forms/triage_questions.py:69]: should use constants

File: applications/helpers/check_your_answers.py (hunk 1 above)
Comment (stephanieboyd) [reply applications/helpers/check_your_answers.py:259]: Added

File: apply_for_a_licence/forms/triage_questions.py (hunk 4 above)
Comment (stephanieboyd) [reply apply_for_a_licence/forms/triage_questions.py:69]: Changed

File: applications/helpers/check_your_answers.py (hunk 1 above)
Comment (stephanieboyd) [reply applications/helpers/check_your_answers.py:259]: Moved

File: applications/helpers/check_your_answers.py (hunk 3 above)
Comment (stephanieboyd) [reply applications/helpers/check_your_answers.py:132]: Changed

File: apply_for_a_licence/forms/triage_questions.py (hunk 4 above)
Comment (stephanieboyd) [reply apply_for_a_licence/forms/triage_questions.py:69]: Changed

**Pull request merged**